gui/wxpython/tplot/frame.py

```diff
@@ -651,14 +651,14 @@ def _writeCSV(self, x, y):
             zipped = list(zip(x, *y))
         else:
             zipped = list(zip(x, y))
-        with open(self.csvpath, "wb") as fi:
+        with open(self.csvpath, "w", newline='') as fi:
```

Comment:
Any specific reason for newline=''? From Py docs:

> When writing output to the stream, if newline is None, any '\n' characters written are translated to the system default line separator, os.linesep. If newline is '' or '\n', no translation takes place

Reply:
To avoid '\r' char see https://docs.python.org/3/library/csv.html#id3

> If newline='' is not specified, newlines embedded inside quoted fields will not be interpreted correctly, and on platforms that use \r\n linendings on write an extra \r will be added. It should always be safe to specify newline='', since the csv module does its own (universal) newline handling.